packages/dev/serializers/src/USDZ/USDZSerializer.ts

```diff
@@ -0,0 +1,732 @@
+                "mirror", // MirroredRepeatWrapping
+            ];
+
+            const repeat = new Vector2(texture.wrapU === Texture.CLAMP_ADDRESSMODE ? 0 : 1, texture.wrapV === Texture.CLAMP_ADDRESSMODE ? 0 : 1);
+            const offset = new Vector2(texture.uOffset, texture.vOffset);
+            const rotation = texture.wAng ?? 0;
+
+            // rotation is around the wrong point. after rotation we need to shift offset again so that we're rotating around the right spot
+            const xRotationOffset = Math.sin(rotation);
+            const yRotationOffset = Math.cos(rotation);
+
+            // texture coordinates start in the opposite corner, need to correct
+            offset.y = 1 - offset.y - repeat.y;
+
+            // turns out QuickLook is buggy and interprets texture repeat inverted/applies operations in a different order.
+            // Apple Feedback: 	FB10036297 and FB11442287
+            if (quickLookCompatible) {
+                // This is NOT correct yet in QuickLook, but comes close for a range of models.
+                // It becomes more incorrect the bigger the offset is
+                offset.x = offset.x / repeat.x;
+                offset.y = offset.y / repeat.y;
+
+                offset.x += xRotationOffset / repeat.x;
+                offset.y += yRotationOffset - 1;
+            } else {
+                // results match glTF results exactly. verified correct in usdview.
+                offset.x += xRotationOffset * repeat.x;
+                offset.y += (1 - yRotationOffset) * repeat.y;
+            }
+
+            return `
+            def Shader "PrimvarReader_${mapType}"
+            {
+                uniform token info:id = "UsdPrimvarReader_float2"
+                float2 inputs:fallback = (0.0, 0.0)
+                token inputs:varname = "${uv}"
+                float2 outputs:result
+            }
+            def Shader "Transform2d_${mapType}"
+            {
+                uniform token info:id = "UsdTransform2d"
+                token inputs:in.connect = </Materials/Material_${material.uniqueId}/PrimvarReader_${mapType}.outputs:result>
+                float inputs:rotation = ${(rotation * (180 / Math.PI)).toFixed(USDZExport.Precision)}
+                float2 inputs:scale = ${USDZExport._BuildVector2(repeat)}
+                float2 inputs:translation = ${USDZExport._BuildVector2(offset)}
+                float2 outputs:result
+            }
+            def Shader "Texture_${texture.uniqueId}_${mapType}"
+            {
+                uniform token info:id = "UsdUVTexture"
+                asset inputs:file = @textures/Texture_${id}.png@
+                float2 inputs:st.connect = </Materials/Material_${material.uniqueId}/Transform2d_${mapType}.outputs:result>
+                ${color !== undefined ? "float4 inputs:scale = " + USDZExport._BuildColor4(color) : ""}
+                token inputs:sourceColorSpace = "${"sRGB" /* TODO CHECK THIS */}"

```

Comment:
Use `texture.gammaSpace` to select between "sRGB" and "raw"

Reply:
Can you explain this more?

Reply:
`gammaSpace` is set to true when the texture is in gamma space (which is the same thing than sRGB space for us).

"raw" means linear space => I read what they have to say about raw / sRGB in https://openusd.org/release/spec_usdpreviewsurface.html to make sure our `gammaSpace` property is the right one to use in this case.